src/services/projectService.ts

```diff
@@ -53,6 +54,9 @@ export default class ProjectService implements IProjectService {
                     project.id = shortid.generate();
                 }
 
+                // automate this eventually
+                project.version = "2.0.0-preview.1";
```

Comment:
https://stackoverflow.com/questions/45978230/get-version-number-from-package-json-in-react-redux-create-react-app